src/components/FileUpload.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { getStorage, ref, uploadBytesResumable, getDownloadURL } from 'firebase/storage';
+import { storage } from '../firebase';
+
+const FileUploader = ({ onUploadComplete }) => {
+  const [file, setFile] = useState(null);
+  const [uploadProgress, setUploadProgress] = useState(0);
```

Comment:
Upon completion of the upload/send, we need to remove the staged upload progress and the file name from the UI as well.

Reply:
After send, the following should look blank:
<img width="338" alt="Screenshot 2024-03-04 at 10 08 15 AM" src="https://github.com/murphy-charitable-foundation/magic-chat-app-web/assets/31979020/2495c16a-fc45-40f4-973b-716b7cc87614">

Reply:
Actually, I wasn't able to see where my attachment went. Are you sure this works?

Reply:
still to check